model/host/host.go

```diff
@@ -1420,29 +1423,6 @@ func (h *Host) clearRunningTaskWithFunc(doUpdate func(update bson.M) error) erro
 	return nil
 }
 
-// UpdateRunningTask updates the running task in the host document and logs an
-// event indicating that it's been assigned a task.
-func (h *Host) UpdateRunningTask(t *task.Task) error {
```

Comment:
Since this is now deleted, we can put the logic in `updateRunningTaskWithFunc` inline into `UpdateRunningTaskWithContext`.